INSTRUMENTS/LABJACK/LABJACKU3/LABJACKU3.py

```diff
@@ -1,4 +1,4 @@
-from flojoy import flojoy, OrderedPair, DataContainer
+from flojoy import flojoy, OrderedPair
 from typing import Optional
 import LabJackPython
```

Comment:
This import seems never accessed.

Reply:
Yes can I import U3 which is a function of the Library LabJackPython without importing it ? Or maybe the good syntax is :
from LabJackPython import u3 ?

Reply:
`import u3` is suffice IMO